src/io/flutter/android/AndroidEmulator.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.android;
+
+import com.intellij.execution.ExecutionException;
+import com.intellij.execution.configurations.GeneralCommandLine;
+import com.intellij.execution.process.OSProcessHandler;
+import com.intellij.execution.process.ProcessAdapter;
+import com.intellij.execution.process.ProcessEvent;
+import com.intellij.execution.process.ProcessOutputTypes;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.util.Key;
+import com.intellij.openapi.vfs.VirtualFile;
+import io.flutter.FlutterMessages;
+
+public class AndroidEmulator {
+  private static final Logger LOG = Logger.getInstance(AndroidEmulator.class);
+
+  final AndroidSdk androidSdk;
```

Comment:
Aside: we've got a mix of field naming conventions (some w/ the `my` prefix); maybe worth a style wiki entry?